tests/foreman/maintain/test_backup_restore.py

```diff
@@ -448,7 +448,12 @@ def test_positive_puppet_backup_restore(
     result = sat_maintain.cli.Backup.run_backup(
         backup_dir=subdir,
         backup_type=backup_type,
-        options={'assumeyes': True, 'plaintext': True, 'skip-pulp-content': False},
+        options={
+            'assumeyes': True,
+            'plaintext': True,
+            'skip-pulp-content': False,
+            'split-pulp-tar': '50M',
+        },
```

Comment:
I see that `split-pulp-tar` option is already covered in `test_positive_backup_split_pulp_tar` and `test_positive_backup_all` tests. So we don't really need to cover it here again, we can just add BZ number to the existing test cases.

Reply:
Oh nice I didn't think to check again after switching the name!